package.json

```diff
@@ -108,7 +108,7 @@
     "webpack": "1.13.1",
     "webpack-dev-middleware": "^1.6.1",
     "webpack-hot-middleware": "^2.12.0",
-    "workerjs": "^0.1.1",
+    "workerjs": "github:jlongster/workerjs",
```

Comment:
@YuichiNukiyama you probably don't have Git installed. I'm curious why you're commenting on a specific diff about a dependency though.